subprojects/packagefiles/libgpiod/meson.build

```diff
@@ -0,0 +1,126 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright 2021 IBM Corp.
+
+project('libgpiod', 'c', 'cpp',
```

Comment:
Hmm, I think you do not actually need a C++ compiler, unless the cxx bindings option is turned on.

So you could remove this from the project options and instead use https://mesonbuild.com/Reference-manual.html#add_languages

Reply:
I also forgot to add in the `meson_version` constraint.

Reply:
Fixed it to use `add_languages('cpp')` in the cxx binding `meson.build`.